assets/js/services/helpers.test.js

```diff
@@ -11,6 +12,17 @@ it("should escape double quotes", () => {
   expect(expected).toMatch(actual);
 });
 
+it("should validate URL pattern", () => {
+  expect(isUrlValid("htttp://northwestern.edu")).toBe(false);
```

Comment:
This is a valid url (without the `www`).  Can we adjust the regex to allow domains of this type?

Reply:
This is invalid as it reads `htttps` and not `https`, there's an extra `t`, otherwise, it should be valid

Reply:
Ooops, I'm blind.  Sorry:)